pkg/controller/jobs/appwrapper/appwrapper_controller.go

```diff
@@ -74,6 +74,9 @@ func init() {
 // +kubebuilder:rbac:groups=kueue.x-k8s.io,resources=resourceflavors,verbs=get;list;watch
 // +kubebuilder:rbac:groups=kueue.x-k8s.io,resources=workloadpriorityclasses,verbs=get;list;watch
 
```

Comment:
```suggestion
```

I'm leaning toward aligning with the existing other Job cases: https://github.com/kubernetes-sigs/kueue/blob/main/pkg/controller/jobs/raycluster/raycluster_controller.go#L58